frontend/src/components/FAQs.tsx

```diff
@@ -0,0 +1,342 @@
+import plusIcon from "../assets/plusIcon.svg";
+import { Accordion } from "../components/index.ts";
+import styles from "../stylesheets/Home.module.css";
+
+import { useState } from "react";
+
+export function FAQs() {
+  const [expandAllCertificationInfo, setExpandAllCertificationInfo] = useState(false);
+  const [expandAllPathwaysInfo, setExpandAllPathwaysInfo] = useState(false);
+
+  const toggleExpandAllCertificationInfo = () => {
+    setExpandAllCertificationInfo(!expandAllCertificationInfo);
+  };
+  const toggleExpandAllPathwaysInfo = () => {
+    setExpandAllPathwaysInfo(!expandAllPathwaysInfo);
+  };
+
+  return (
+    <>
+      <div className={styles.textDivider2}>
+        <div className={styles.heading}>FAQ's</div>
+      </div>
+
+      <div className={styles.FaqsContainer}>
+        <div className={styles.infoTitle}>
+          <h2 className={styles.red}>Certification Information</h2>
+          <button className={styles.expand} onClick={toggleExpandAllCertificationInfo}>
+            <span style={{ marginLeft: 20 }}>
+              {expandAllCertificationInfo ? "Collapse" : "Expand All"}
+            </span>
+            <div className={styles.expandAllPlus}>
+              <img src={plusIcon} style={{ height: "14px", width: "14px" }}></img>
+            </div>
+            {expandAllCertificationInfo}
+          </button>
+        </div>
+        <hr
+          style={{ border: "1px solid #d8d8d8", boxShadow: "20px 2px 4px rgba(0, 0, 0, 0.1)" }}
+        ></hr>
+        <Accordion
+          title="Why should I become a CID?"
+          toggleAll={expandAllCertificationInfo}
+          content={
+            <>
+              <p>
+                Certified Interior Designer (CID) is the exclusive legally recognized title for
+                interior designers in California. CIDs demonstrate their education, experience, and
+                examination expertise in California, reducing legal risks and boosting client
+                confidence. They are accredited for professionalism, mandated to adhere to the{" "}
+                <a href="https://ccidc.org/code-of-ethics/CCIDC" style={{ textDecoration: "none" }}>
+                  <span className={styles.lightBlue}>Code of Ethics and Conduct.</span>
+                </a>
+              </p>
+            </>
```

Comment:
Move this inside the `<accordion> (here) </Accordian>` and you won't need the <> fragments to wrap the content anymore either. Sent more details to Jared.